
Turn 1
================================================================
error: failed to decode bot message: "T" (at position 1)
Last input received was:
Traceback (most recent call last):
  File "MyBot.py", line 143, in <module>
    enemy_ships = fetch_enemy()
  File "MyBot.py", line 106, in fetch_enemy
    if player.id != me.id:
AttributeError: 'int' object has no attribute 'id'

